naga/tests/snapshots.rs

```diff
@@ -886,6 +886,12 @@ fn convert_spv_all() {
         true,
         Targets::METAL | Targets::GLSL | Targets::HLSL | Targets::WGSL,
     );
+    convert_spv(
+        "unnamed-gl-per-vertex",
+        true,
```

Comment:
Not sure if the `adjust_coordinate_space` should be `true`, but it might not matter either way for this test case?

Reply:
Yeah, I think any value should be fine.